dot/core/service_integration_test.go

```diff
@@ -589,6 +590,7 @@ func TestService_HandleSubmittedExtrinsic(t *testing.T) {
 
 	extBytes := createExtrinsic(t, rt, genHeader.Hash(), 0)
 
+	// Tests use old extrinsic version currently
```

Comment:
Should we leave it as is? 🤔

Reply:
ya I think so. My current understanding is not that this PR is to migrate to newest runtime, but more to add logic to support without breaking development to then migrate further based on the 9.20 branch.

That being said, I do have a newer runtime that I am using to test maintainTxnPool, so if you think its appropriate i can create another test function that uses the latest runtime here too. This could be a good call

Reply:
Update: Talked with Tim about this and we are going to leave as is and test in latest Runtime branch